internal/app/options/flags.go

```diff
@@ -130,6 +130,72 @@ func GetFlagsAndBeforeFunc() ([]cli.Flag, cli.BeforeFunc) {
 			Usage:   "specify the path of the config file",
 		},
 		cli.VersionFlag,
+		altsrc.NewIntFlag(&cli.IntFlag{
+			Name:  "num-namespaces",
+			Usage: "maximum number of namespaces that can be copied. Recommended to keep this number under 15 to avoid performance issues. Defaults to 8.",
```

Comment:
Let's call it "cosmos-max-namespaces" and clarify in usage that it's strictly related to Cosmos DB

Reply:
Also this one shouldn't be hidden I think